tox.ini

```diff
@@ -32,9 +32,6 @@ deps =
 commands =
   ansibledevel: sh -c "{envpython} -m pip show ansible | >/dev/null grep '^Version: 2.10.0$'"
   ansibledevel: sh -c "{envpython} -m pip show ansible-base | >/dev/null grep '^Version: 2.10.0.dev0$'"
-  # Collections outside of ACD that config/routing.yml points to in Core:
```

Comment:
This doesn't fix anything currently because the problem is on the install stage, not in commands.